build_kernel4.6.0.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "	****Cheking for requirements****"
+# requirements Ubuntu
+sudo apt-get install build-essential gcc-arm-linux-gnueabi libglib2.0-dev zlib1g-dev
+
+echo "	****Getting the source****"
+# get the git repo
+https://github.com/Xilinx/linux-xlnx.git
```

Comment:
updated in [https://github.com/apertus-open-source-cinema/axiom-beta-qemu/pull/3](url)